source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -70,211 +92,267 @@ Mixing
+   ``rpmbuild``, etc. to build them. If they are not built on Clear,
+   make sure your configuration and toolchain builds them correctly for Clear, or there is no guarantee they will be compatible.
 
-      # sudo mixer init-mix -all -clearver 13180 -mixver 10
+#. **Import RPMs into workspace**. Create an :file:`rpms` directory in your
+   workspace (for example :file:`/home/clr/mix/rpms`), and copy the RPMs you
+   want into that directory. Next, add the following to your
+   :file:`builder.conf`:
 
-#. **Create/locate RPMs for mix.**. (Steps 4-6 are necessary only if you
-   want to add your own RPMs to the Mix. If you are working only with Clear
-   bundles, then skip to Step 7.)
+   .. code-block:: bash
 
-   If you are creating RPMs from scratch, you may use ``autospec``,
-   ``mock``, ``rpmbuild``, etc. to build them. If they are not
-   built on Clear, make sure your configuration and toolchain builds them correctly for Clear, or there is no guarantee
-   they will be compatible.
+      RPMDIR=/home/clr/mix/rpms
 
-#. **Import RPMs into workspace**. The way to do this is to create an
-   ``rpms`` directory in your workspace (for example ``/home/clr/mix/rpms``),
-   and to copy the RPMs you want into that directory. The mixer script will
-   look here for RPMs in order to build a local RPM repo for yum to use.
+   Mixer will look in this directory for RPMs to build a local RPM repo for
+   yum to use.
 
 #. **Create a local RPM repo**. Create an empty directory in your workspace
-   named ``local`` and add the paths in your builder.conf::
+   named :file:`local` and add the path in your :file:`builder.conf`:
 
-    RPMDIR=/home/clr/mix/rpms
-    REPODIR=/home/clr/mix/local
+   .. code-block:: bash
 
-    These variables are automatically read; you simply need to run::
+      REPODIR=/home/clr/mix/local
+
+   Once these values are configured, you can generate the yum repo by
+   running the following command:
+
+   .. code-block:: bash
 
       # sudo mixer add-rpms
 
-   After the script exits, you should see your RPMs and a repodata directory in
-   ``/home/clr/mix/local``. If the RPMs are not all in the local directory, check
-   to make sure that they are indeed valid RPM files and not corrupt.
+   After the tool exits, you should see your RPMs and a repodata directory in
+   :file:`/home/clr/mix/local`. If the RPMs are not all in this :file:`local`
+   directory, check to make sure that they are indeed valid RPM files and not
+   corrupt.
+
+#. **Update/Add bundle definitions**. You can easily add bundles to your mix
+   by running:
+
+   .. code-block:: bash
 
-#. **Update/Add bundle definitions**. The mixer uses a local clone of the
-   ``clr-bundles`` repo to define bundles for the mix.
+      # sudo mixer bundle add bundle1,bundle2,...
+
+   This command copies the specified bundle defintion files from your
+   configured upstream version of Clear Linux (:file:`.clearversion`) into
+   your :file:`mix-bundles` directory.
```

Comment:
I don't see the "mix-bundles" directory in my testing.  I see a "mixbundles" file which contains a list of bundles.

Reply:
Again, yes, while these docs have been valid for the last month, _new changes_ have been made in that time that make much of this outdated. I'm updating the documentation as we speak.